src/__screenshot_tests__/list-screenshot-test.tsx

```diff
@@ -182,3 +182,134 @@ test.each(controlsWithIconButton)('Rows with %s using big fontSize', async (cont
     const image = await list.screenshot();
     expect(image).toMatchImageSnapshot();
 });
+
+test('Rows over inverse background', async () => {
+    await openStoryPage({
+        id: 'components-lists--row-list-story',
+        device: 'MOBILE_IOS',
+        args: {
+            overInverse: true,
+        },
+    });
+
+    const list = await screen.findByTestId('list');
+    const image = await list.screenshot();
+    expect(image).toMatchImageSnapshot();
+});
+
+test('BoxedRows over inverse background', async () => {
+    await openStoryPage({
+        id: 'components-lists--boxed-row-list-story',
+        device: 'MOBILE_IOS',
+        args: {
+            overInverse: true,
+        },
+    });
+
+    const list = await screen.findByTestId('list');
+    const image = await list.screenshot();
+    expect(image).toMatchImageSnapshot();
+});
+
+test('BoxedRows inverse', async () => {
+    await openStoryPage({
+        id: 'components-lists--boxed-row-list-story',
+        device: 'MOBILE_IOS',
+        args: {
+            isInverse: true,
+        },
+    });
+
+    const list = await screen.findByTestId('list');
+    const image = await list.screenshot();
+    expect(image).toMatchImageSnapshot();
+});
+
+test('BoxedRows inverse over inverse', async () => {
+    await openStoryPage({
+        id: 'components-lists--boxed-row-list-story',
+        device: 'MOBILE_IOS',
+        args: {
+            isInverse: true,
+            overInverse: true,
+        },
+    });
+
+    const list = await screen.findByTestId('list');
+    const image = await list.screenshot();
+    expect(image).toMatchImageSnapshot();
+});
```

Comment:
These tests aren't related with this task. I've included some screenshot tests for the inverse/overInverse cases that weren't covered yet